Repositories/UserRepository.cs

```diff
@@ -22,8 +22,9 @@ public async Task<IEnumerable<User>> GetAll()
         {
             try
             {
-                List<User> userCollection = await GetCollection().AsQueryable().ToListAsync();
-                return userCollection.Where(x => x.Deleted == false);
+                List<User> userCollection = await GetCollection().AsQueryable()
+                    .Where(u => !u.Deleted).ToListAsync();
```

Comment:
@christopherplummer Doing this on the database server now instead of the API server. Think it should be pretty fast with the index I added.